scripts/deep_learning/hierarchy/LaterTier/TrainLaterTierModel_showers.ipynb

```diff
@@ -0,0 +1,897 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe1752bb",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "###########################################################\n",
+    "# Imports\n",
+    "###########################################################\n",
```

Comment:
Notebooks permit you to define markdown cells (e.g. see the vertex network notebook). Prefer the use of those markdown cells to add heading sections to the workflow and any description of what's going on and what's required. In this specific case, no need to add a comment block to delineate imports.